plugins/source/aws/client/services.go

```diff
@@ -375,3 +256,249 @@ type Services struct {
 	Workspaces                   services.WorkspacesClient
 	Xray                         services.XrayClient
 }
+
+func (s *Services) InitService(awsConfig *aws.Config, service string) {
+	c := awsConfig.Copy()
+	switch strings.ToLower(service) {
+	case "accessanalyzer":
```

Comment:
Not necessarily blocking, but I think it would be great if we could expose constants for all the supported services, which could then be used by the resolvers rather than hardcoded strings. This would also make discovery of where the names come from easier for developers using IDEs